test/e2e/main.go

```diff
@@ -20,6 +22,14 @@ type Test struct {
 func main() {
 	logger := log.New(os.Stdout, "test-e2e", log.LstdFlags)
 
+	latestVersion, err := testnet.GetLatestVersion()
+	testnet.NoError("failed to get latest version", err)
+
+	// TODO: remove me when the issue with bbr is fixed on k8s
+	latestVersion = "v2.1.2"
```

Comment:
Due to the limitation on the cluster, the BBR does not work at the moment. So a hard coded app version that works with it, is added here temporarily to make the test work.